modules/apps/frontend-theme-locos/locos-theme/src/css/_banner.scss

```diff
@@ -0,0 +1,93 @@
+%astyle {
+  color: #999990;
+  font-family: 'Poppins', sans-serif;
+}
+%ahover {
+  text-decoration: none;
+  color: #f28959;
+  border-bottom: 3px solid #f28959;
+  -webkit-transition:border-bottom .5s ease;
+  -moz-transition:border-bottom .5s ease;
+  -ms-transition:border-bottom .5s ease;
+  -o-transition:border-bottom .5s ease;
+  transition:border-bottom .5s ease;
+}
+
+#sign-in {
+  order: 3;
+  font-size: 20px;
+  margin: 20px 20px 25px 20px;
+  @extend %astyle;
+  &:hover {
+    @extend %ahover;
+  }
+}
+
+#banner {
+  overflow: hidden;
+  display: flex;
+  justify-content: space-between;
+  flex-flow: row wrap;
+  align-items: flex-end;
+  align-content: center;
+  padding: 0px 10px 0px 10px;
+  background-color: #144d52;
+  z-index: 5;
```

Comment:
Make sure ```z-index``` match the values in https://github.com/liferay/liferay-portal/blob/7.0.x/modules/apps/foundation/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util.js#L1862...L1874.